apps/pie-storybook/stories/pie-checkbox-group.stories.ts

```diff
@@ -1,36 +1,97 @@
 import { html } from 'lit';
-
+import { ifDefined } from 'lit/directives/if-defined.js';
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-checkbox-group';
-import { CheckboxGroupProps } from '@justeattakeaway/pie-checkbox-group';
+import { CheckboxGroupProps, defaultProps, statusTypes } from '@justeattakeaway/pie-checkbox-group';
 /* eslint-enable import/no-duplicates */
 
 import { type StoryMeta } from '../types';
 import { createStory } from '../utilities';
 
+import '@justeattakeaway/pie-link';
+import '@justeattakeaway/pie-checkbox';
+
 type CheckboxGroupStoryMeta = StoryMeta<CheckboxGroupProps>;
 
-const defaultArgs: CheckboxGroupProps = {};
+const defaultArgs: CheckboxGroupProps = {
+    ...defaultProps,
+};
 
 const checkboxGroupStoryMeta: CheckboxGroupStoryMeta = {
     title: 'Checkbox Group',
     component: 'pie-checkbox-group',
-    argTypes: {},
+    argTypes: {
+        name: {
+            description: 'The name associated with the group.',
+            control: 'text',
+        },
+        label: {
+            description: 'The visible label for the checkbox group.',
+            control: 'text',
+        },
+        status: {
+            description: 'The status of the checkbox group component / assistive text. Can be default, success or error.',
+            control: 'select',
+            options: statusTypes,
+            defaultValue: {
+                summary: defaultProps.status,
+            },
+        },
+        disabled: {
+            description: 'If true, disables the whole checkbox group. If you need to disable only one checkbox pass the disabled prop to the needed checkbox.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.disabled,
+            },
+        },
+        assistiveText: {
+            description: 'An optional assistive text to display below the checkbox group.',
+            control: 'text',
+            defaultValue: {
+                summary: '',
+            },
+        },
+    },
     args: defaultArgs,
     parameters: {
         design: {
             type: 'figma',
-            url: '',
+            url: 'https://www.figma.com/design/aD4m0j97Ruw8Q4S5lED2Bl/Checkbox-audit?node-id=9938-14138&t=1V1VEYmtArFih6AX-4',
```

Comment:
```suggestion
            url: 'https://www.figma.com/file/aD4m0j97Ruw8Q4S5lED2Bl/Checkbox-audit?node-id=9938-14138&t=1V1VEYmtArFih6AX-4',
```

Reply:
Interesting, for me the https://www.figma.com/file/aD4m0j97Ruw8Q4S5lED2Bl/Checkbox-audit?node-id=9938-14138&t=1V1VEYmtArFih6AX-4 link redirects to https://www.figma.com/design/aD4m0j97Ruw8Q4S5lED2Bl/Checkbox-audit?node-id=9938-14138&t=1V1VEYmtArFih6AX-4

Reply:
`/design/` seems fine to me

Reply:
but it doesn't work for me
<img width="1457" alt="Screenshot 2024-07-18 at 11 17 51" src="https://github.com/user-attachments/assets/9e60fcdd-15bf-49ab-8c0c-28bb45bab05a">

Reply:
Oh, I see, good catch 👍